app/service/task_status_service.rb

```diff
@@ -71,6 +72,12 @@ def support_contact_details_status
     :not_started
   end
 
+  def receive_csv_status
+    return :completed if @form.submission_type == "email_with_csv"
```

Comment:
Is this a bit more rails-y?

```suggestion
    return :completed if @form.email_with_csv?
```